src/components/Pages/SetupPage/SetupPage.tsx

```diff
@@ -125,6 +162,15 @@ export const SetupPage = () => {
           }
         }
 
+        getAnalytics().track(TrackingEvents.TERMS_OF_SERVICE_SUCCESS, {
+          // eslint-disable-next-line @typescript-eslint/naming-convention
+          eth_address: accountRef.current,
+          // eslint-disable-next-line @typescript-eslint/naming-convention
+          is_guest: false,
+          email: email || undefined,
```

Comment:
email can be an empty string at this point. Sending it as undefined to prevent issues.